### ISSUE TITLE
Training doubts

### ISSUE THREAD

**Issue by kaiser24** [pjreddie/darknet#1985]
Hello,

I'm trying to detect 4 classes of vehicles (bykes, cars, buses, and trucks) seen from above and I have a few questions about training with a custom data set for the best results.

should I label every single vehicle that appears on the images I use to train?
I extract the images from videos, so I wonder if i leave some cars or bykes without label would affect my results?

also. reading this guy's tutorial
https://github.com/AlexeyAB/darknet#how-to-improve-object-detection
he says : "desirable that your training dataset include images with non-labeled objects that you do not want to detect - negative samples without bounded box (empty .txt files) - use as many images of negative samples as there are images with objects"

Does that mean that i should include a lot of images of empty streets?

Thanks in advance.

**Comment by giorgi3092** [comment]
That is a good question. I want to know too. 
As https://github.com/qqwweee/keras-yolo3/issues/294#issuecomment-495923451
this comment says, that is exactly what you should do. 
Did you figure out anything?
